src/App.jsx

```diff
@@ -1,10 +1,92 @@
 import './App.css'
+import axios from "axios";
+import {useEffect, useState} from "react";
+import PokemonCard from "./components/pokemonCard.jsx";
 
 function App() {
+    const [pokemonId, setPokemonId] = useState([])
```

Comment:
Ik vind de variabelnaam niet helemaal passen bij wat het eigenlijk is. Het is volgens mij een array van pokemons, dus dan zou ik ook een variabelnaam verwachten die daar beter bij past:
```suggestion
    const [pokemons, setPokemons] = useState([]);
```